buildchain/buildchain/docker_command.py

```diff
@@ -22,6 +22,29 @@
 DOCKER_CLIENT : docker.DockerClient = docker.from_env()
 
 
+def task_errors(
+        *caught_errors: Type[Exception]
+) -> Callable[[Any], Any]:
+    """Wrap a callable to create a resilient `doit` task
+
+    This decorator wraps action functions in a try…except block that abstracts
+    the exceptions raised by the underlying actions (docker API calls, file
+    system actions…) and returns a result conforming to `doit`'s expectations:
```

Comment:
"conforming to doit's expectation", I think you need to explain a bit more what you want here, because an exception is conform to doit's expectation

> if it raises any exception, it will be considered an error

What we want here is to enforce the use of `TaskError` in case of error to have a nicer output?